addons/website_slides/static/src/js/slides_course_quiz_question_form.js

```diff
@@ -123,6 +125,18 @@ var QuestionFormWidget = publicWidget.Widget.extend({
         });
     },
 
+    /**
+     * Handler when a user clicks on the 'Save & New' button.
```

Comment:
Also, need to change docstring of `_validateQuestion` and `_cancelValidation`.

Reply:
Hello @ibo-odoo :wave: ,
I believe they're already appropriate. :slightly_smiling_face: 
But if you still want me to change, it, please let me know what would sound suitable to you.

Reply:
@shut-odoo How can that be appropriate. In `_validateQuestion` It should be `Save & Close` and In `_cancelValidation` It should be `Discard` not cancel.

Reply:
@ibo-odoo 
Really sorry, I missed the nuances here  :sweat_smile: . I will be careful next time.
I have updated the code :+1: